tests/test_core.py

```diff
@@ -5682,6 +5685,7 @@ def test_zlib(self):
                 includes=[path_from_root('tests', 'zlib'), 'building', 'zlib'],
                 force_c=True)
 
+  @needs_make('make')
   @is_slow_test
```

Comment:
bullet can run using cmake, but i guess it still uses make afterwards?

Reply:
CMake just generates makefiles (or Ninja files). I guess if we added ninja support to our cmake builds it could work. I just observe that zlib is attempting to run mingw32-make. Which, I guess I should update that string, actually.